packages/labs/ssr-react/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "@lit-labs/ssr-react",
+  "version": "0.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Lit SSR integration for React",
+  "license": "BSD-3-Clause",
+  "author": "Google LLC",
+  "homepage": "https://github.com/lit/lit/tree/main/packages/labs/ssr-react",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lit/lit.git",
+    "directory": "packages/labs/ssr-react"
+  },
+  "main": "index.js",
+  "typings": "index.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./index.d.ts",
+      "node": "./node/index.js",
+      "default": "./index.js"
+    },
+    "./enable-lit-ssr.js": {
+      "types": "./enable-lit-ssr.d.ts",
+      "node": "./node/enable-lit-ssr.js",
+      "default": "./enable-lit-ssr.js"
+    },
+    "./jsx-dev-runtime": {
```

Comment:
Add .js extension

Reply:
We can't do that as the compilers that do the transform don't add the `.js` extension on the import line. See https://www.typescriptlang.org/play?jsx=4#code/PQKhAIAECsGcA8CSBbADgewE4BcDK6BXTAYwFMoAbAS2wFoKBDAI1mFlk1s1IeO3BDAAUAB5kAT1qkKpZKQB2-VJnSoAjAF4A5MtVqt4ANalx24+K0A+IQAsqQgN4BtNQBoATK4DMAXQB0yAyoABRU4BqW4CKwqAzyRiYaDlQAvpbJKSJssfKWAJQposASUjJyipZAA